### DISCUSSION TITLE
Notification when not receiving XP for a certain time

### DISCUSSION THREAD

**Discussion by Zdub55** [runelite/runelite#16424] [Ideas]
Howdy! 

Could not find this idea anywhere, a simple plugin that provides you with a notification when you stop receiving XP.

**Comment by Nerdpuff** [comment]
Would the `Idle Notifier` plugin help with this? It has a setting for `Idle Interaction Notifications`.

https://github.com/runelite/runelite/wiki/Idle-Notifier
